content/docs/iac/concepts/vs/cloud-template-transpilers/aws-cdk/_index.md

```diff
@@ -34,6 +34,41 @@ Choosing the right infrastructure as code tool is important, and we want you to
 
 AWS Cloud Development Kit (CDK) is an open-source software development framework for defining AWS cloud resources with general-purpose programming languages. CDK is a transpiler that converts program code into AWS CloudFormation JSON/YAML templates and other assets and then submits them to AWS and the CloudFormation deployment service.
 
+## Why Choose Pulumi over AWS CDK?
+
+Pulumi has various benefits over AWS CDK across dimensions of speed, security, savings, and scale.
```

Comment:
Do we have any case studies we can highlight to land these strengths with proof points?

Reply:
I would also embellish this a bit more:

```suggestion
Pulumi and CDK share a heritage of embracing general-purpose programming languages for infrastructure as code. Pulumi has various benefits over AWS CDK across dimensions of speed, security, savings, and scale. Also, because of Pulumi's inherently multi-language and multi-cloud nature, Pulumi is able to do things that CDK cannot, such as deploy to Azure or Snowflake alongside AWS. Having a truly multi-language runtime also enables Pulumi to deliver a much more seamless and productive developer experience, with the many end users who made the switch claiming better performance, deployment times, and greater developer joy.
```

Reply:
Yes, we can layer in case studies